columnflow/plotting/plot_util.py

```diff
@@ -279,12 +282,12 @@ def prepare_plot_config(
             "kwargs": {
                 "norm": data_norm,
                 "label": "Data",
-                "yerr": False if data_hide_errors[i] else None,
-            },
-            "ratio_kwargs": {
-                "norm": h_mc.values() * data_norm / mc_norm,
-                "yerr": False if data_hide_errors[i] else None,
+                "yerr": False if data_hide_errors[0] else None,
```

Comment:
How about "skip year als long as some data process requested it?"
```suggestion
                "yerr": False if any(data_hide_errors) else None,
```